RegExp.make.js

```diff
@@ -971,11 +971,12 @@ RegExp.make = (function () {
   }
 
   function make(flags, template, ...values) {
+    // make is always bound to a flags string
     if ('string' === typeof template && values.length === 0) {
       // Allow RegExp.make(i)`...` to specify flags.
       // This calling convention is disjoint with use as a template tag
       // since the typeof a template record is 'object'.
-      return make.bind(null, template /* use as flags instead */);
+      return make.bind(null, flags + template /* use as flags instead */);
```

Comment:
Ah, you're right, [§21.2.3.2.2#7](http://www.ecma-international.org/ecma-262/6.0/#sec-regexpinitialize) says

> *if it contains the same code unit more than once, throw a `SyntaxError` exception.*

I didn't consider that. Do you think we should throw immediately? Filter out duplicates?

The change just looked simple and possible useful. I don't want to add any complexity…